teams/vsp/teams/tools/backend/triage/ongoing-support-developer.md

```diff
@@ -96,6 +96,33 @@ For quick reference these are also pinned in the [#vsp-backend-utility-dev] chan
 
 - add pull request to the [Code Review Epic][Super Epics] for the current sprint
 
+#### Overriding Jumbo Pull Requests
+
+When a pull request is large enough, the `Danger-bot` status check will fail. 
+
+![image](https://user-images.githubusercontent.com/3077884/90051737-257f4100-dca6-11ea-9e7a-dd0cb6b1c178.png)
+
+When the pull request is not cognitively complex and cannot be easily broken up into smaller pull requests, it is appropriate to override this status check. 
+
+**Prerequisites**
+
+1. Ensure you are a repo admin
+
+![image](https://user-images.githubusercontent.com/3077884/90051558-da652e00-dca5-11ea-88f5-75d39d0453c7.png)
+
+**Steps**
+
+1. Navigate to `master` branch protection rules
+
+![image](https://user-images.githubusercontent.com/3077884/90052019-9161a980-dca6-11ea-87c4-2000aab5ca13.png)
+
+2. Disable `danger/danger` status check
```

Comment:
I think an additional warning like "Note: this disables the danger bot for all PRs and should be reverted promptly after the PR is merged" might be helpful on this step.  I know I can be prone to being halfway through some steps and getting distracted by Slack or something.
Or maybe somewhere before the steps that explains "This is how you turn off danger bot long enough to merge the PR..." because I didn't really realize what these steps were recommending until I got to the end.